src/styles/getTooltipStyles.js

```diff
@@ -161,20 +166,15 @@ export default function getTooltipStyles(
         pointerWrapperStyle: {
             position: 'fixed',
 
-            // By default, the pointer's top-left will align with the top-left of the wrapped tooltip.
+            // By default, the pointer's top-left will align with the top-left of the tooltip wrapper.
             //
             // To align it vertically, we'll:
+            //   If the pointer should be below the tooltip wrapper, shift the pointer down (+) by the tooltip height,
+            //   so that the top of the pointer lines up with the bottom of the tooltip
             //
-            //   Shift the pointer up (-) by component's height, so that the bottom of the pointer lines up
-            //   with the top of the wrapped component.
-            //
-            //   OR if it should show below:
-            //
-            //   Shift the pointer down (+) by the component's height,
-            //      so that the top of the pointer aligns with the bottom of the component.
-            //
-            // Always add the manual vertical shift passed in as a parameter.
-            top: shouldShowBelow ? (manualShiftVertical - POINTER_HEIGHT) : (tooltipHeight + manualShiftVertical),
+            //   OR if the pointer should be above the tooltip wrapper, then the pointer up (-) by the pointer's height
+            //   so that the bottom of the pointer lines up with the top of the tooltip
+            top: shouldShowBelow ? -POINTER_HEIGHT : tooltipHeight,
```

Comment:
Here we removed `manualShiftVertical` can you explain why it's not needed?

Reply:
@alex-mechler just waiting for this

Reply:
Cc: @tgolen

Reply:
Hi! What can I help with?

Reply:
Good catch @Santhosh-Sellavel. I think `manualShiftVertical` wasn't really working so I removed it for testing, but I'm going to re-investigate this and see if it's needed or not and if I can fix it.

Reply:
Oh, got it this was just on the pointer. You can test but `manualShiftVertical` seems to work as expected:

|||
|---|---|
|4|<img width="725" alt="image" src="https://user-images.githubusercontent.com/47436092/211122358-2cefaf59-fbcb-48f4-a1fd-70f288bbb56d.png">|
|24|<img width="726" alt="image" src="https://user-images.githubusercontent.com/47436092/211122647-db965c4e-fd08-4906-b6ee-ef9cf4027e1d.png">|
|-24|<img width="712" alt="image" src="https://user-images.githubusercontent.com/47436092/211122716-36bda6bc-b749-4424-9fde-b08167ec6bec.png">|